README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/reark/reark.svg?branch=master)](https://travis-ci.org/reark/reark)
```

Comment:
Good idea to add this, but I think the most common placement would be under the main header, right?

I think we should move the latest updates anyway to a separate changelog.